graylog2-server/src/main/java/org/graylog2/streams/OutputAVImpl.java

```diff
@@ -32,44 +32,52 @@
 @JsonAutoDetect
 @CollectionName("outputs")
 public abstract class OutputAVImpl implements Output {
```

Comment:
Exactly. We need to keep it so plugins are potentially able to work with Outputs.